deepchem/models/tensorgraph/tensor_graph.py

```diff
@@ -52,6 +52,7 @@ def __init__(self,
     learning_rate: float or LearningRateSchedule
       the learning rate to use for optimization
     kwargs
+      "configproto": a tf.ConfigProto() object used to create tf.Session()
```

Comment:
Probably better to make `configproto` an explicit keyword arg instead of adding onto `kwargs`.